README.md

```diff
@@ -35,8 +35,19 @@ Here are some useful links to start:
 * Localization happens on
   [Pontoon](https://pontoon.mozilla.org/projects/mozilla-vpn-client/).
 
+### Pre-commit formatting hook
+
 If you want to submit a pull-request, please, install the clang format
-pre-commit hook: `./scripts/git-pre-commit-format install`
+pre-commit hook that lints code.
```

Comment:
We need to write the pre-commit hook script. Am i wrong?

Reply:
I looked at this PR, and understand what you meant. I neglected to add this as step 3 before. This has been fixed, thanks.